src/iotjs_binding.c

```diff
@@ -21,9 +21,11 @@
 #include <string.h>
 
 
-static iotjs_jval_t jglobal;
-static iotjs_jargs_t jargs_empty;
-
+static iotjs_jargs_t jargs_empty = {.unsafe = { 0, 0, NULL },
+#ifndef NDEBUG
+                                    .flag_create = IOTJS_VALID_MAGIC_SEQUENCE
```

Comment:
`.unsafe` and `.flag_create` looks indentation wrong.

Reply:
`tools/check_tidy --autoedit` made this indentation. What would be the correct indentation?

Reply:
I see. It's right.